### ISSUE TITLE
Webpage doesn't work

### ISSUE THREAD

**Issue by janisozaur** [cegui/cegui#1203]
Steps to follow:
* Notice the link in the readme pointing to http://www.cegui.org.uk/

What happens:
* Browser goes to http://www.cegui.org.uk/
  - This triggers a warning from HTTPS everywhere about using legacy HTTP
* After allowing single connection to insecure webpage, user gets redirected to page with invalid certificates
  - `The certificate is only valid for the following names: *.kasserver.com, kasserver.com`, but the address I was redireccted to is https://cegui.org.uk/

Expected result:
* Readme contains proper link, website uses valid certificate

**Comment by manhnt9** [comment]
Yeah someone should renew the certificate and use letsencrypt instead, which is a free cert and support automatic renewal.
